### ISSUE TITLE
Date fields in Cross Industry Invoices (CII) are not validated

### ISSUE THREAD

**Issue by JannickWeisshaupt** [itplr-kosit/validator-configuration-xrechnung#56]
Date fields in Cross Industry Invoices (CII) are not validated and can therefore contain invalid dates (see [attached invoice](https://github.com/itplr-kosit/validator-configuration-xrechnung/files/5808369/cii_with_bad_date.zip) Line 14)

The visualization needs correct dates in order to convert them into the German format and therefore fails in this case.

Versions:
Validator: 1.4.0
Configuration: 2.0.0

**Comment by phax** [comment]
The test file example values are:
```xml
<udt:DateTimeString format="102">00000000</udt:DateTimeString>
```

**Comment by rkottmann** [comment]
It's a CEN business rule which does to get checked by https://github.com/ConnectingEurope/eInvoicing-EN16931/. @phax may I ask you to report it there?

**Comment by phax** [comment]
Cite from the EN 16931-3:
> The the mapping of the SDM only code “102” (YYYYMMDD) shall be used. In a scenario with applied extensions the two additional formats 610 (YYYYMM) and 616 (YYYYWW) can be used as well.

So the missing rules would be
* Length must be 8
* All chars must be numeric
* year must be > 0
* Month must be >= 1 and <= 12
* Day must be >= 1 and <= 31

The check for the exact days per month doesn't seem reasonable for Schematron

**Comment by phax** [comment]
Fixed in EN version 1.3.5. Will be included in the next validator configuration.
